src/request/import-words/ImportWordsApi.js

```diff
@@ -0,0 +1,79 @@
+/* global PopupApi */
+/* global RecoveryWordsInput */
+/* global PassphraseInput */
+/* global MnemonicPhrase */
+/* global PrivacyAgent */
+/* global Nimiq */
+/* global Key */
+/* global EncryptionType */
+/* global KeyStore */
+class ImportWordsApi extends PopupApi {
+    constructor() {
+        super();
+
+        // start UI
+        this._makeView();
+    }
+
+    async onRequest() {
+        // show UI
+        window.location.hash = ImportWordsApi.Pages.PRIVACY_AGENT;
+    }
+
+    _makeView() {
+        const $rootElement = /** @type {HTMLElement} */ (document.getElementById('app'));
+        const $privacyAgent = /** @type {HTMLElement} */ ($rootElement.querySelector('#privacy'));
+        const $enterWords = /** @type {HTMLElement} */ ($rootElement.querySelector('#words'));
+        const $enterPassphrase = /** @type {HTMLElement} */ ($rootElement.querySelector('#passphrase'));
+
+        const privacyAgent = new PrivacyAgent();
+        privacyAgent.on(PrivacyAgent.Events.CONFIRM, () => {
+            window.location.hash = ImportWordsApi.Pages.ENTER_WORDS;
+        });
+        $privacyAgent.appendChild(privacyAgent.getElement());
+
+        const recoveryWordsInput = new RecoveryWordsInput();
+        // for debugging
+        // window.input = recoveryWordsInput;
+        recoveryWordsInput.on(RecoveryWordsInput.Events.COMPLETE, this._onRecoveryWordsEntered.bind(this));
+        $enterWords.appendChild(recoveryWordsInput.getElement());
+
+        const passphraseInput = new PassphraseInput(true);
+        passphraseInput.on(PassphraseInput.Events.PASSPHRASE_ENTERED, this._handlePassphraseInput.bind(this));
+        $enterPassphrase.appendChild(passphraseInput.getElement());
```

Comment:
Does the user also have to repeat the passphrase?

Reply:
ooookaaaay :P